internal/storage/fake_storage_util.go

```diff
@@ -26,6 +26,33 @@ const TestObjectName string = "gcsfuse/default.txt"
 const ContentInTestObject string = "Hello GCSFuse!!!"
 const TestObjectGeneration int64 = 780
 
+type FakeStorage interface {
+	CreateStorageHandle() (storageHandleObj StorageHandle)
```

Comment:
can you rename storageHandleObj to sh here as well.

Reply:
fixed